modules/openapi-generator/src/main/resources/JavaSpring/converter.mustache

```diff
@@ -0,0 +1,34 @@
+package org.openapitools.configuration;
+
+{{#models}}
+    {{#model}}
+        {{#isEnum}}
+import {{modelPackage}}.{{name}};
+        {{/isEnum}}
+    {{/model}}
+{{/models}}
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
+
+@Configuration
+public class EnumConverters {
```

Comment:
I would rename it to `EnumConverterConfiguration`

Reply:
Additional the config is just working if you import it manually, activate component scan or create a auto config file. Documentation would be good, what is expected to work

Reply:
I've changed the name.
What kind of documentation is expected? I suppose the scan is done as it is done for others `@Configuration`no?